test/countries/test_canada.py

```diff
@@ -56,6 +56,21 @@ def test_islander_day(self):
             self.assertNotIn(dt + relativedelta(days=-1), pei_holidays)
             self.assertNotIn(dt + relativedelta(days=+1), pei_holidays)
 
+    def test_yukon_heritage_day(self):
+        # https://www.timeanddate.com/holidays/canada/heritage-day-yukon
+        yt_holidays = holidays.CA(subdiv="YT")
+        for dt in [
```

Comment:
> Not that important here and mostly for future consistency: please use () instead of [] where possible.

There was lists in similar cases in file, so I made a list too, not tuple.